generators/python/sdk/versions.yml

```diff
@@ -1,4 +1,11 @@
 # For unreleased changes, use unreleased.yml
+- version: 4.15.2
```

Comment:
If we consider this a feature, let's bump the minor version. If it's a fix, then the version is good as-is.